installer/client/src/pages/WelcomeScreen.vue

```diff
@@ -0,0 +1,100 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template>
+  <oxd-form class="orangehrm-installer-page">
+    <oxd-text tag="h5" class="orangehrm-installer-page-title">
+      Welcome to OrangeHRM Stater Version 5.0 Setup Wizard
```

Comment:
```suggestion
      Welcome to OrangeHRM Stater Version {{ productVersion }} Setup Wizard
```
shall we get this from BE

Reply:
Sure, that will be better